src/raftstore/coprocessor/region_info_accessor.rs

```diff
@@ -226,7 +226,10 @@ impl RegionCollector {
         // receive an `Update` message, the region may have been deleted for some reason. So we
         // handle it according to whether the region exists in the collection.
         if self.regions.contains_key(&region.get_id()) {
-            info!("region_collector: trying to create region {} but it already exists, try to update it", region.get_id());
+            info!(
+                "region_collector: trying to create region but it already exists, try to update it";
```

Comment:
```suggestion
                "trying to create this region but it already exists, try to update it";
```

Looks like we dont need to print `region_collector: ` as it is already printed by the file name.

Reply:
Is `region_collector` a key word for the monitor?